packages/app/src/resource/BotEditor.tsx

```diff
@@ -106,16 +100,15 @@ export function BotEditor(): JSX.Element | null {
       e.stopPropagation();
       setLoading(true);
       try {
-        const code = await getCodeOutput();
-        await medplum.post(medplum.fhirUrl('Bot', id, '$deploy'), { code });
+        await medplum.get(medplum.fhirUrl('Bot', id, '$deploy'));
```

Comment:
Prefer to keep this a `POST` request.

Reply:
We can't do post without a body in `MedplumClient`. Should I make that possible instead?

Reply:
I just passed in `undefined` instead

Reply:
Update 2:
Just passing `undefined` kinda broke the `MedplumClient.post` functionality, I ended up having to add support for `POST` without a body. Works now 👍